### DISCUSSION TITLE
Downloadable template for virtual tour example.

### DISCUSSION THREAD

**Discussion by isaacullah** [mistic100/Photo-Sphere-Viewer#1485] [General]
Firstly, many thanks for your work on this great tool. I've been using Marzipano to create virtual tools, but would like to migrate to Photo Sphere Viewer now that Marzipano seems like a dead project. I'm able to follow your examples here to create a simple viewer, but am having trouble building a working virtual tour. I am wondering if you can make the tour example available to download as a template from which I can build out my own pages? I guess I have been spoiled by the Marizipano-Tool that provides a full template with all resources bundled that I can go in and edit further as necessary. Many thnaks!

**Comment by mistic100** [comment]
https://github.com/mistic100/Photo-Sphere-Viewer/blob/main/examples/plugin-virtual-tour.html

But it is basically the same thing as the demo. PSV is a library not a final product, so will still have to build arround it.

If you have any problem please create a "question" issue with your specific problem and a working demo what of you have so far.

**Comment by isaacullah** [comment]
Thank you!

On Fri, Nov 15, 2024 at 12:11 AM Damien Sorel ***@***.***>
wrote:

> Closed #1485
> <https://github.com/mistic100/Photo-Sphere-Viewer/discussions/1485> as
> resolved.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mistic100/Photo-Sphere-Viewer/discussions/1485>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACDYS5D4YEKMALYDDTPQHBD2AWUDLAVCNFSM6AAAAABRZ5UHLKVHI2DSMVQWIX3LMV45UABFIRUXGY3VONZWS33OIV3GK3TUHI5E433UNFTGSY3BORUW63R3GE3DCMJXGY3A>
> .
> You are receiving this because you authored the thread.Message ID:
> <mistic100/Photo-Sphere-Viewer/repo-discussions/1485/discussion_event/1611766
> @github.com>
>


-- 
___________________________________________________________________________
Isaac I. Ullah, PhD
*Recent papers:*
Laborscapes and Archaeologies of Sustainability:
http://dx.doi.org/10.1558/jma.39327
Water, dust, and loess: co-evolution of landscapes, farming, and human
society: https://doi.org/10.1016/j.jaa.2019.101067
Modeling feedbacks between human and natural processes in the land system:
 https://doi.org/10.5194/esd-9-895-2018
<https://doi.org/10.5194/esd-9-895-2018>
<https://doi.org/10.5194/esd-9-895-2018>
